### ISSUE TITLE
Draft with invalid data returns invalid result

### ISSUE THREAD

**Issue by DianaLease** [accordproject/cicero#551]
**Describe the bug**
If a template expects an integer for a variable, passing a string that cannot be coerced into a valid intege, results in a draft result with `NaN` in the text.

**To Reproduce**
1. Run `cicero draft --data data.json --template src/acceptance-of-delivery/` where `data.json` is the following...
 ```
{
  "clauseId": "89ce3388-6765-4a6b-b9c8-cce05321aa2a",
  "shipper": "Party A",
  "receiver": "Party B",
  "deliverable": "Widgets",
  "businessDays": "a",
  "attachment": "Attachment X",
  "$class": "org.accordproject.acceptanceofdelivery.AcceptanceOfDeliveryClause"
}
```

2. See result

```
Acceptance of Delivery.
----

"Party A" will be deemed to have completed its delivery obligations
if in "Party B"'s opinion, the "Widgets" satisfies the
Acceptance Criteria, and "Party B" notifies "Party A" in writing
that it is accepting the "Widgets".

Inspection and Notice.
----

"Party B" will have NaN Business Days to inspect and
evaluate the "Widgets" on the delivery date before notifying
"Party A" that it is either accepting or rejecting the
"Widgets".

Acceptance Criteria.
----

The "Acceptance Criteria" are the specifications the "Widgets"
must meet for the "Party A" to comply with its requirements and
obligations under this agreement, detailed in "Attachment X", attached
to this agreement.
```

**Expected behavior**
If the template expects an integer and is given a string that cannot be coerced into a valid integer, draft should return an error.

**Comment by jeromesimeon** [comment]
I think just a consequence of the current validation semantics for Concerto. See https://github.com/accordproject/concerto/issues/157

This kind of surprising result should be addressed when we switch to the stricter validation semantic decided for the next version of Concerto.

**Comment by jeromesimeon** [comment]
@DianaLease Fixed in the Cicero `0.22` with the lovely strict validation from Concerto `1.0`! 😃

```
zsh-5.8$ more data.json 
{
 "clauseId": "89ce3388-6765-4a6b-b9c8-cce05321aa2a",
 "shipper": "Party A",
 "receiver": "Party B",
 "deliverable": "Widgets",
 "businessDays": "a",
 "attachment": "Attachment X",
 "$class": "org.accordproject.acceptanceofdelivery.AcceptanceOfDeliveryClause"
}
zsh-5.8$ cicero draft --data data.json --template src/acceptance-of-delivery
10:57:16 AM - ERROR: Expected value "a" to be of type Long
zsh-5.8$ cicero --version
0.22.0
zsh-5.8$ 
```
